_rules/SC1-3-5-autocomplete-valid.md

```diff
@@ -0,0 +1,177 @@
+---
+name: Autocomplete valid
+description: |
+  This rule checks that the HTML autocomplete attribute has a correct value
+
+success_criterion:
+- 1.3.5 # Identify Input Purpose
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Wilco Fiers
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any HTML `input`, `select` and `textarea` element with a [non-empty](#non-empty) `autocomplete` attribute, except if one of the following is true:
+
+- The element is **not** [visible on the page](#visible-on-the-page) or [exposed to assistive technologies](#exposed-to-assistive-technologies)
+- The element is an `input` element a `type` of `hidden`, `button`, `submit` or `reset`
```

Comment:
Should there be a "with" after "input element" before "a type"

Reply:
Should be "with a `type` of ..."